text/0000_require_minimal_ruby_version.md

```diff
@@ -0,0 +1,38 @@
+- Feature Name: mandatory_required_ruby_version
+- Start Date: 2020-04-13
+- RFC PR: 
+- Bundler Issue: 
+
+# Summary
+
+Calling `spec.required_ruby_version` in a gemspec should be mandatory.
+
+# Motivation
+
+My current problem is that many gems are not publishing their `required_ruby_version`.
+
+This means that a Gemfile/gemspec must know what is the maximum version of a gem for a given version of Ruby, instead of bundler figuring it out for us.
+
+It takes about one minute for the gem author to add the line. When the author doesn't, everyone that uses the gem, directly or indirectly, may have to figure out the maximum version and specify it in their own gemfiles.
+
+Note that there is currently no good way to correct this mistake. Once it is done, the only option [is to yank the gem](https://github.com/rubygems/rubygems/issues/1506#issuecomment-188472423) which is typically not what one wants to do.
```

Comment:
It does resolve the linked comment in not making it possible in the first place (forgetting to specify it), but it absolutely does *not* resolve the linked comment in general as errors can still be made by a wrong specification (e.g. too low a minimal version) and I hope that it will be possible one day to fix this kind of error.